src/Migrator/Factory.php

```diff
@@ -193,7 +193,7 @@ public function runDown(Model $entity, $database, $pretend = false)
 
         $migrator->setConnection($database);
         $migrator->setEntity($entity);
-        $migrator->rollback($this->getMigrationPath(), ['pretend' => (bool) $pretend]);
+        $migrator->rollback($this->getMigrationPath(), $pretend);
```

Comment:
This code is calling Laravel own `Migrator` which expect an array, not a boolean.

[Migrator::rollback](https://github.com/laravel/framework/blob/5.3/src/Illuminate/Database/Migrations/Migrator.php#L179)